install_config/topics/applying_configuration_changes.adoc

```diff
@@ -18,6 +18,17 @@ configuration changes, see xref:../install_config/master_node_configuration#mast
 # systemctl restart atomic-openshift-node
 ----
 
+[NOTE]
+====
+Kubernetes architecture expects reliable endpoints from cloud providers. When a
+cloud provider is down, the kubelet prevents {product-title} from restarting.
+If the underlying cloud provider endpoints are not reliable, do not install a
+cluster using cloud provider integration. Install the cluster as if it was in
+a no-cloud environment. It is not recommended to toggle cloud provider
+integration on or off in an installed cluster. However, if that scenario is
+unavoidable then follow the steps outlined below.
```

Comment:
unavoidable,

s/the steps outlined below./complete the following process.

>! below adj
Do not use to indicate a relative location in a document, as in “the information below.” Use “the information
later in this section.”
Alternatively, you can use “following” to refer to a table, to art, or to a list. Example: “The following table
shows ...”